system/bin/Detach

```diff
@@ -221,7 +221,7 @@ rm -f "$MAGMOD/apps_lists.txt"
 
 # Adding app
 add() {
-SERVICESH=$MAGMOD/service.sh
+#SERVICESH=$MAGMOD/service.sh
```

Comment:
The script can't run propeply as the var for the `services.sh` file is declarated for each functions to be sure that everyting will working in any situation.